contracts/cw721-base/src/query.rs

```diff
@@ -143,17 +143,20 @@ where
         let start = start_after.map(Bound::exclusive);
 
         let owner_addr = deps.api.addr_validate(&owner)?;
-        let pks: Vec<_> = self
+        let res = self
             .tokens
             .idx
             .owner
             .prefix(owner_addr)
             .keys(deps.storage, start, None, Order::Ascending)
-            .take(limit)
+            .take(limit);
+
+        let tokens = res
+            .into_iter()
+            .flatten()
```

Comment:
This is incorrect.
By flattening, you just filter out all Err() variants and unwrap the Ok() variant: https://doc.rust-lang.org/src/core/result.rs.html#1648-1664

We want an error if any fail, not silent ignoring